tests/integration/test_sns.py

```diff
@@ -44,21 +43,27 @@
 TEST_LAMBDA_ECHO_FILE = os.path.join(THIS_FOLDER, "lambdas", "lambda_echo.py")
 
 
-class SNSTest(unittest.TestCase):
-    @classmethod
-    def setUpClass(cls):
-        cls.sqs_client = aws_stack.create_external_boto_client("sqs")
-        cls.sns_client = aws_stack.create_external_boto_client("sns")
-        cls.topic_arn = cls.sns_client.create_topic(Name=TEST_TOPIC_NAME)["TopicArn"]
-        cls.queue_url = cls.sqs_client.create_queue(QueueName=TEST_QUEUE_NAME)["QueueUrl"]
-        cls.dlq_url = cls.sqs_client.create_queue(QueueName=TEST_QUEUE_DLQ_NAME)["QueueUrl"]
+@pytest.fixture(scope="class")
+def setup(request):
```

Comment:
Thanks for updating the PR @pinzon . Actually, I think my comment regarding the `class` scoped fixture may have been a bit misleading - over time we may slightly restructure this test to avoid global/static state and instead make the individual test method atomic/self-contained.

In any case, this will be a bigger refactoring that we'll tackle once we start introducing parallelized test execution. For now, this looks great - thanks again for the refactoring! 🎉